src/i18n/TextCollection/i18nTextCollector.php

```diff
@@ -920,6 +926,49 @@ public function collectFromEntityProviders($filePath, Module $module = null)
         return $entities;
     }
 
+    /**
+     * Extracts translations for ORM fields
+     *
+     * @param string $filePath
+     * @return array
+     */
+    public function collectFromORM($filePath)
+    {
+        $entities = [];
+        $classes = ClassInfo::classes_for_file($filePath);
+        foreach ($classes as $class) {
+            // Skip non-implementing classes
+            if (!class_exists($class ?? '')) {
```

Comment:
Is the null coalesce needed here? Can it be anything other than a string provided from `ClassInfo::classes_for_file`?

Reply:
removed